docs/mkdocs.yml

```diff
@@ -26,6 +26,13 @@ extra_css:
 #       redirect_maps:
 #         tutorials/faq: config/troubleshooting/faq.md
 
+# We do not use `mike`, but enabling this will enable the version selector UI.
+# It references `versions.json` on `gh-pages` branch,
+# however we have a basic setup that only matches `version` to a subdirectory.
+extra:
+  version:
+    provider: mike
```

Comment:
I think i stumbled over a section in the docs where it says if one changes `extra:` one might add others extras as the default values got overwritten by specifying custom ones. I will see if i find it again

Reply:
This is following the instructions from `mkdocs-material` on enabling their `mike` integration.